tests/sqllogictests/suites/base/05_ddl/05_0010_ddl_create_udf.test

```diff
@@ -22,6 +22,18 @@ CREATE FUNCTION IF NOT EXISTS isnotempty_with_desc AS(p) -> not(is_null(p)) DESC
 statement error 2603
 CREATE FUNCTION isnotempty_with_desc AS(p) -> not(is_null(p)) DESC = 'This is a description'
 
+statement ok
+CREATE FUNCTION with_lambda AS(list) -> array_filter(list, x -> x > 42)
```

Comment:
Need tests to validate whether the new function works or not.

Reply:
nice, I didn't think of that